tests/backup/cluster_controller_test.go

```diff
@@ -0,0 +1,104 @@
+package tests
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/portworx/torpedo/drivers/backup/controllers/cluster"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+// This testcase demonstrates the various functionalities of the ClusterController
+var _ = Describe("{ClusterControllerDemo}", func() {
+	var (
+		testRailId           int
+		clusterControllerMap map[string]*cluster.ClusterController
+	)
+
+	JustBeforeEach(func() {
+		testRailId = 31313
+
+		Step("Add test case cluster controllers to cluster-controller-map", func() {
+			err := cluster.AddTestCaseClusterControllers(&clusterControllerMap, testRailId)
+			log.FailOnError(err, "failed to add test case cluster controllers to cluster-controller-map")
+		})
+	})
+
+	It("Cluster Controller Demo", func() {
+
+		Step("Schedule each application on a unique namespace in source-cluster", func() {
+			for _, appKey := range Inst().AppList {
+				namespace := fmt.Sprintf("%s-app-namespace-step-1", appKey)
+
+				err := clusterControllerMap["source-cluster"].Application(appKey).ScheduleOnNamespace(namespace)
+				log.FailOnError(err, fmt.Sprintf("failed to schedule the application %s on the namespace %s", appKey, namespace))
+
+				err = clusterControllerMap["source-cluster"].Namespace(namespace).Application(appKey).Validate()
```

Comment:
Change the methods to `SelectNamespace()` and `SelectApplication()`

Reply:
Done.